### ISSUE TITLE
replacing scda and scda.2022 useage by rcd sme

### ISSUE THREAD

**Issue by shajoezhu** [insightsengineering/teal.modules.hermes#374]

**Comment by zdz2101** [comment]
@shajoezhu @edelarua looks like [PR](https://github.com/insightsengineering/teal.modules.hermes/pull/312/files) handled the the decoupling of it, however `data-raw/data.R` still mentions `library(scda)` but it is also technically already in the `.Rbuildignore` ? Is it supposed to be swapped appropriate with RCD call and assure everything still runs correctly?

**Comment by edelarua** [comment]
@zdz2101 yes, I would replace it there as well so there's no mention of it anywhere in the package.
